src/plugins/intel_cpu/src/nodes/bin_conv.cpp

```diff
@@ -636,6 +636,9 @@ struct jit_uni_bin_conv_kernel_f32 : public jit_uni_bin_conv_kernel, public jit_
         if (jcp_.with_binarization) {
             int binarization_idx = p.find(primitive_kind::binarization);
 
+            if (binarization_idx < 0)  // for coverity correctness
+                binarization_idx = 0;
```

Comment:
`binarization_idx < 0` clearly indicates that the kind of post op is missed. Suggest using IE_ASSERT(binarization_idx>=0) with an error message that `binarization` post op hasn't been found.

Reply:
What about rewriting this like this:
```c++
const int binarization_idx = p.find(primitive_kind::binarization);
if (jcp.with_binarization && binarization_idx != -1) {
    ...;
}
```

Reply:
But it looks like `jcp.with_binarization` indicates that binarization is expected and if it is absent in the post ops list, then it is probably an exceptional situation and we throw? What do you think?

Reply:
Done in first variant

Reply:
@EgorDuplensky It's codegen. What kind of heaviness are you talking about?